Il semblerait que $\delta_0$ ne soit pas la mesure qui maximise $\mathbb E(TMRCA)$.
Regardons ça de plus près.

Prenons $n=8$, alors,

$$\mathbb E(TMRCA) = \sum_{k=2}^{8} \frac{2}{k(k-1)} = 2\left(1 - \frac{1}{8}\right) = \frac{14}{8} = 1.75$$

A présent trouvons une mesure qui donne une valeur plus grande.
Soit $\Lambda = \delta_{1/4}$.
Alors,

$$
\lambda_{n,k} = \int_0^1 x^{k-2}(1-x)^{n-k} d\Lambda(x) = \left(\frac{1}{4}\right)^{k-2} \left(1 - \frac{1}{4}\right)^{n-k} = \left(\frac{1}{4}\right)^{k-2} \left(\frac{3}{4}\right)^{n-k}
$$


$$
\lambda_n = \sum_{k=2}^{n} \binom{n}{k} \lambda_{n,k} = \sum_{k=2}^{n} \binom{n}{k} \left(\frac{1}{4}\right)^{k-2} \left(\frac{3}{4}\right)^{n-k} =  16\left(1 -\left(\frac34\right)^n - \frac{n}{4}\left(\frac34\right)^{n-1}\right)
$$


$$\mathbb E(TMRCA) = \mathbb E(\tau |N_0 = n)  = \frac{1}{\lambda_n} + \sum_{k=2}^{n} \frac{\binom{n}{k} \lambda_{n,k}}{\lambda_n} \mathbb E(\tau | N_0 = n-k+1 )$$

On a $\mathbb E(\tau | N_0 = 1) = 0, \mathbb E(\tau | N_0 = 2) = 1$.

In [11]:
import numpy as np
import math

In [12]:
n=8
lmb_n_k = lambda n, k : 0.25**(k-2) * (0.75)**(n - k) 
lmb_n = lambda n : 16 * (1 - (0.75)**n - n/4 * (0.75)**(n-1))
print(lmb_n(n))

10.126708984375


In [13]:
Esperance = [0, 1]
for i in range(3, n+1):
    En = 1/lmb_n(i) + np.sum([
        math.comb(i,k)*(lmb_n_k(i,k))/lmb_n(i) * Esperance[i-k+1 -1]  for k in range(2,i)
    ])
    Esperance.append(En)

In [14]:
Esperance

[0,
 1,
 np.float64(1.3),
 np.float64(1.465671641791045),
 np.float64(1.5737535725627185),
 np.float64(1.6516884637144404),
 np.float64(1.7116689366662232),
 np.float64(1.759966277592896)]

In [19]:
Esperance[-1] > 1.75

np.True_

ChatGPT a trouvé ça : 
Solving ((*)) successively for (b=2,\dots,8) yields the **exact rational values**

\begin{aligned}  
\lambda_2&=\frac{1}{1}  \\
\lambda_3&=\frac52  \\
\lambda_4&=\frac{67}{16}  \\
\lambda_5&=\frac{47}{8}  \\
\lambda_6&=\frac{1909}{256}  \\
\lambda_7&=\frac{4547}{512}  \\
\lambda_8&=\frac{41479}{4096} \\
T_{\delta_{1/4}}(2)&=\frac{1}{1}\quad  \\
T_{\delta_{1/4}}(3)&=\frac{13}{10}\quad  \\
T_{\delta_{1/4}}(4)&=\frac{491}{335}\quad  \\
T_{\delta_{1/4}}(5)&=\frac{19823}{12596}\  \\
T_{\delta_{1/4}}(6)&=\frac{39716111}{24045764}\quad  \\
T_{\delta_{1/4}}(7)&=\frac{467867967601}{273340222270}\  \\
T_{\delta_{1/4}}(8)&=\frac{19954284839411683}{11337879079537330}\approx 1.7599662776\ldots
\end{aligned}  
